app/models/foreman_tasks/task/dynflow_task.rb

```diff
@@ -85,15 +85,16 @@ def get_humanized(method)
 
     def self.consistency_check
       fixed_count = 0
+      logger = Foreman::Logging.logger('foreman-tasks')
       self.running.each do |task|
         begin
           changes = task.update_from_dynflow(task.execution_plan.to_hash)
           unless changes.empty?
             fixed_count += 1
-            Rails.logger.warn("Task %s updated at consistency check: %s" % [task.id, changes.inspect])
+            logger.warn("Task %s updated at consistency check: %s" % [task.id, changes.inspect])
           end
         rescue => e
-          Rails.logger.warn("Failed at consistency check for task %s: %s\n %s" % [task.id, e.message, e.backtrace.join("\n")])
+          logger.warn("Failed at consistency check for task %s: %s\n %s" % [task.id, e.message, e.backtrace.join("\n")])
```

Comment:
Could you also search for this kind of exception logging and convert that to use https://github.com/theforeman/foreman/blob/develop/lib/foreman/logging.rb#L60 ?